packages/build/src/telemetry/complete.js

```diff
@@ -13,53 +12,63 @@ const { track } = require('./track')
 
 // Send telemetry request when build completes
 const trackBuildComplete = async function ({
+  status,
   commandsCount,
   netlifyConfig,
   durationNs,
   siteInfo,
   telemetry,
-  mode,
   testOpts,
 }) {
   if (!telemetry) {
     return
   }
 
-  const payload = getPayload({ commandsCount, netlifyConfig, durationNs, siteInfo, mode })
+  const payload = getPayload({ status, commandsCount, netlifyConfig, durationNs, siteInfo })
   await track({ payload, testOpts })
 }
 
 // Retrieve telemetry information
-const getPayload = function ({ commandsCount, netlifyConfig, durationNs, siteInfo: { id: siteId }, mode }) {
-  const durationMs = roundTimerToMillisecs(durationNs)
-  const plugins = Object.values(netlifyConfig.plugins).map(getPluginPackageName)
-  return {
-    type: 'track',
-    event: 'netlifyCI:buildComplete',
-    options: {},
-    userId: null,
-    anonymousId: uuidv4(),
-    meta: {
-      timestamp: Date.now(),
-    },
+const getPayload = function ({ status, commandsCount, netlifyConfig, durationNs, siteInfo }) {
+  const basePayload = {
+    event: 'build:ci_build_process_completed',
+    timestamp: Date.now(),
     properties: {
+      status,
       steps: commandsCount,
-      duration: durationMs,
-      pluginCount: plugins.length,
-      plugins,
-      siteId,
-      isCI,
-      isNetlifyCI: mode === 'buildbot',
       buildVersion: version,
       nodeVersion: nodeVersion.replace('v', ''),
       osPlatform: OS_TYPES[platform],
       osName: osName(),
     },
   }
+
+  return addDuration(addConfigData(addSiteInfoData(basePayload, siteInfo), netlifyConfig), durationNs)
+}
+
+const addSiteInfoData = function (payload, siteInfo = {}) {
+  const userData = siteInfo.user_id ? { userId: siteInfo.user_id } : { anonymousId: uuidv4() }
+  const properties = { properties: { ...payload.properties, siteId: siteInfo.id } }
+  return { ...userData, ...payload, ...properties }
 }
 
-const getPluginPackageName = function ({ packageName }) {
-  return packageName
+const addConfigData = function (payload, netlifyConfig = {}) {
+  if (netlifyConfig.plugins === undefined) return payload
+
+  // Only extract package name and its origin
+  const plugins = Object.values(netlifyConfig.plugins).map(({ package: packageName, origin }) => ({
+    name: packageName,
+    origin,
+  }))
+  const properties = { properties: { ...payload.properties, plugins, pluginCount: plugins.length } }
+  return { ...payload, ...properties }
+}
+
+const addDuration = function (payload, durationNs) {
+  if (typeof durationNs !== 'number') return payload
+
+  const durationMs = roundTimerToMillisecs(durationNs)
+  return { ...payload, ...{ properties: { ...payload.properties, duration: durationMs } } }
```

Comment:
```suggestion
  return { ...payload, properties: { ...payload.properties, duration: durationMs } }
```

Reply:
Nice catch, thansk! The out datted diff unfortunately ended up lot letting me commit this 😂 addressed in https://github.com/netlify/build/pull/2349/commits/cbef07a8f78fa796aea32bb6ef391783f06f2369